common/element_handle.go

```diff
@@ -106,6 +106,7 @@ func getElementHandleActionFn(
 	}
 }
 
+//nolint:funlen,gocognit,cyclop,unparam
```

Comment:
We better refactor this one in TODO [here](https://github.com/grafana/xk6-browser/issues/58#issue-1044893666) instead of silencing it.

Reply:
Can you add a TODO to #58? Also for the other one.

Reply:
Just added a generic "fill all `nolint`s".